### DISCUSSION TITLE
zie/ziewin/3.0/ja/doc/qb/req

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#76] [Q&A]
# zie/ziewin/3.0/ja/doc/qb/req

Z and I Emulator for Windows をサポートしています。 workstation s.をインストールする際には、メモリとストレージの要件だけでなく、ハードウェアとソフトウェアの要件も含めてください。 Z and I Emulator for Windows . 次のセクションでは、モニタ、アダプタ、キーボードのサポートについて説明し、リストします。 ...

https://help.hcl-software.com/zie/ziewin/3.0/ja/doc/qb/req.html

**Comment by KunioMatsuo** [comment]
False Japanese Translation.
Current: インストール予定Z and I Emulator for Windows
Correct: Z and I Emulator for Windows のインストールの計画
